index.html

```diff
@@ -33,5 +33,166 @@ <h1 class="top-name">ELDON RATZLAFF</h1>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+
+
+<!-- proyect 4 -->
+<div class="project4"></div>
+<div class="proyect-foot1">
+    <p class="proyect-foot1-p">Project name goes here</p>
+    <ul class="proyect-foot1-ul">
+        <li class="proyect-foot1-li">HTML/CSS</li>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+
+
+<!-- proyect 5 -->
+<div class="project5"></div>
+<div class="proyect-foot1">
+    <p class="proyect-foot1-p">Project name goes here</p>
+    <ul class="proyect-foot1-ul">
+        <li class="proyect-foot1-li">HTML/CSS</li>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+
+
+
+<!-- proyect 6 -->
+<div class="project6"></div>
+<div class="proyect-foot1">
+    <p class="proyect-foot1-p">Project name goes here</p>
+    <ul class="proyect-foot1-ul">
+        <li class="proyect-foot1-li">HTML/CSS</li>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+</section>
+
+
+<section class="about-section">
+    <div>
+        <h2 class="about-title">About
+        </h2>
+        <p class="about-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+    </div>
+
+    <div class="get-mycv">
+        <h3 class="get-mycv-h">Get my CV</h3>
+        <div class="get-mycv-icon"></div>
+    </div>
+
+
+    <div class="languages">
+       
+        <h3 class="languages-h about-boxes" >Languages</h3>
+        <ul class="languages-ul">
+            <li>JavaScript(ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+        </ul>
+ 
+    </div>
+
+
+    <div class="frameworks">
+        <h3 class="about-boxes">Frameworks</h3>
+        <ul class="languages-ul">
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+        </ul>
+    </div>
```

Comment:
- [ ] As stated in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please use `ul` and `li` for categories in the "About myself" section. `Languages`, `Frameworks`, and `Skills` should be list items. kindly correct that.

```
<ul>
  <li>
     <h3 class="languages-h about-boxes" >Languages</h3>
     . . .
  </li>
  <li>
     <h3 class="languages-h about-boxes" >Languages</h3>
     . . .
  </li>  
  <li>
     <h3  class="about-boxes">Skills</h3>
     . . .
  </li>
</ul>
```

- [ ] Please kindly fix the `margin` and `padding` of this section to match the Figma design

Your design 👇 
![image](https://user-images.githubusercontent.com/78251927/205418719-f1e31c26-b067-4fd9-b19c-bcf2df0896d8.png)

Figma design
![image](https://user-images.githubusercontent.com/78251927/205418684-945792ff-bd28-4f52-a838-3f7027f37f08.png)